strings_lab

```diff
@@ -0,0 +1,153 @@
+// // 1. `console.log` the first character of a string.
```

Comment:
Good demonstration of string properties and methods to solve problems. I like how you used different methods to solve problems and there's no hard coded elements here. Keep up the good work and keep asking me and Alejo questions! ALSO try bonuses next time. I am confident that you can do them and they are also very good practice.